### ISSUE TITLE
Adding entry sensor to Simplisafe Homebridge doesn't see the new sensor

### ISSUE THREAD

**Issue by cwmn** [homebridge-simplisafe3/homebridge-simplisafe3#215]
Just added a new sensor to my Simplisafe system, works fine but doesn't show in Homebridge. Any suggestions?

**Comment by shamoon** [comment]
Hi @cwmn , as the bot stated, there is not nearly enough info here to be able to help. Please provide more details about what happened, what you’ve tried to do to fix it, and logs etc. if appropriate. And, have you restarted it?

**Comment by cwmn** [comment]
Michael,
Sorry I had never seen a firm like that for reporting issues. Must admit I wasn’t sure of the syntax etc. the good news is that I solved my issue with and o do mean my issue not yours. Here’s what I did. 

Went into Homebridge, Accessories, saw the sensor, clicked the gear in the upper right corner, selected show in dashboard widget, everything is working fine now! Problem solved!

Thanks for your plug-in and for reaching out to me. 

Chuck
